CHANGES.rst

```diff
@@ -0,0 +1,7 @@
+Changelog
+=========
+
+0.6.2 (unreleased)
+----------------
```

Comment:
I went with a small bump because these days ZODB3 is nothing but a meta-distribution around the dependencies that are now explicitly listed (with only >= version pins, hmm, that I should probably add back), so the net effect is the same.

But I don't really have a strong opinion one way or the other.